challenges/__tests__/linked-list.test.js

```diff
@@ -69,11 +69,14 @@ describe('testing the linked list module', () => {
     expect(test).toEqual('{ 0 } -> { 1 } -> { 2 } -> { 3 } -> {null}');
   });
 
+<<<<<<< HEAD
```

Comment:
Looks like that there was a possible merge conflict on line 72-79 of your linked list test file, that's causing issues when you try and run jest. Could you please get that resolved and then I will regrade for you, let me know if you have any questions!